### PR TITLE
Add publications 2024

### PR DESCRIPTION
**Pull request by vsoch**
This pull request will add several papers from the last few years, and talks as well! Preview:

image
image
image

Now that we have quite a list, I think it makes sense to break apart a bit by years. Since we don't want single entries standing out, I created groups. This PR superceeds #70 which I don't think was ever reviewed.

### CODE REVIEW

File: publications/Flux-D.Nichols_IPDPS2024.pdf
Comment (bhatele) [inline publications/Flux-D.Nichols_IPDPS2024.pdf:1]: This pdf is not ready to see the light of day. We'll send a new version once the camera-ready is prepared.

Comment (vsoch) [reply publications/Flux-D.Nichols_IPDPS2024.pdf:1]: Alright - will remove it for now, and we can update when the "camera ready" version is available.

Comment (vsoch) [reply publications/Flux-D.Nichols_IPDPS2024.pdf:1]: Done.

**Comment by vsoch** [comment]
@grondo @garlick could we merge this soon? Another one just came in.

**Comment by bhatele** [comment]
We can use this pdf for Daniel's IPDPS paper now: https://www.cs.umd.edu/~bhatele/pubs/pdf/2024/ipdps2024.pdf

**Review by grondo — APPROVED** [review]
LGTM

**Comment by vsoch** [comment]
@grondo I'm just going to add that paper link then we should be good - give me a minute!

**Comment by vsoch** [comment]
okay should be good after CI (and another quick glance). Note that we just had a new paper notice come in, and the Flux Operator was also published (link TBA) so likely I'll do another PR next week with those two.

**Pull request merged**